source/brailleDisplayDrivers/nlseReaderZoomax.py

```diff
@@ -23,13 +23,16 @@
 So each command starts with the escape character.
 When part of the command payload, the escape character is duplicated. """
 
+PROTOCOL_NVDA = b"\x03"
+"""The device has a command 0x15 for setting protocol type, the parameter value 0x03 is specifying that the screen reader is NVDA."""
+
 
 class DeviceCommand(bytes, Enum):
 	DISPLAY_DATA = b"\x01"
 	REQUEST_INFO = b"\x02"
 	REQUEST_VERSION = b"\x05"
 	REPEAT_ALL = b"\x08"
-	PROTOCOL_ONOFF = b"\x15"
+	PROTOCOL_TYPE = b"\x15"
```

Comment:
This is unfortunately an API breaking change. You will either need to create a structure which mimics the name, or keep the old name. Please see [deprecations.md](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/deprecations.md) for more information


e.g.

```suggestion
	PROTOCOL_TYPE = b"\x15"
	
	if NVDAState._allowDeprecatedAPI():
        	@property
        	def PROTOCOL_ONOFF(self) -> bytes:
        		# log warnings here about the deprecation here
        		return self.PROTOCOL_TYPE
		
```

Reply:
@seanbudd 
I reverted to the old name.
Thank you!